test/models/place_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class PlaceTest < ActiveSupport::TestCase
+
+  def new_place 
+    Place.new(place_id: "abc123", 
+                 name: "first one", 
+                 image_url: "www.yeah.com", 
+                 rating: "3.5", 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.